homeassistant/components/honeywell/sensor.py

```diff
@@ -0,0 +1,111 @@
+"""Support for Honeywell (US) Total Connect Comfort sensors."""
```

Comment:
Sensors are new? 

Please leave these out of scope for this PR. PRs should be as minimal as possible. After the config flow has been merged, you can open up a second PR to add sensors.